TTS/tts/models/vits.py

```diff
@@ -221,7 +263,7 @@ def __init__(self, config: Coqpit, speaker_manager: SpeakerManager = None):
         super().__init__(config)
 
         self.END2END = True
-
+        self.audio_config = config["audio"]
```

Comment:
why do you need this ? There is already self.config

Reply:
We need it to know the sample rate of the TTS model to resample the wav or raise an error when the sample rate of the TTS model differs from the speaker encoder sample rate.